RadixWallet/Features/TransactionReviewFeature/SelectFeePayer/SelectFeePayer+View.swift

```diff
@@ -127,22 +119,33 @@ enum SelectAccountToPayForFeeRow {
 		let isSelected: Bool
 		let action: () -> Void
 
+		var buttonState: RadioButton.State {
+			viewState.insufficientBalance ? .disabled : (isSelected ? .selected : .unselected)
+		}
+
 		var body: some SwiftUI.View {
-			Button(action: action) {
-				Card {
-					VStack(spacing: .zero) {
-						AccountCard(kind: .innerCompact, account: viewState.account)
+			VStack(alignment: .leading) {
+				Button(action: action) {
+					Card {
+						VStack(spacing: .zero) {
+							AccountCard(kind: .innerCompact, account: viewState.account)
 
-						HStack {
-							ResourceBalanceView(.fungible(viewState.fungible), appearance: .compact)
+							HStack {
+								ResourceBalanceView(.fungible(viewState.fungible), appearance: .compact)
 
-							RadioButton(appearance: .dark, state: isSelected ? .selected : .unselected)
+								RadioButton(appearance: .dark, state: buttonState)
+							}
+							.padding(.medium3)
 						}
-						.padding(.medium3)
 					}
 				}
+				.buttonStyle(.inert)
+				.disabled(buttonState == .disabled)
+
+				if viewState.insufficientBalance {
+					WarningErrorView(text: L10n.TransactionReview.FeePayerValidation.insufficientBalance, type: .error)
+				}
```

Comment:
There is no official UI no. I tried this way too and I think it looks ugly, but most of all I did it this way because that's how it already looks on the screen that is presenting this screen: 
![image](https://github.com/user-attachments/assets/936fa2d5-825a-4eec-8d2b-e9745451fb94)